tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -738,6 +774,20 @@ public void expectNextElement(ManualSubscriber<T> sub, T expected) throws Interr
       }
     }
 
+    public void expectAnyNextElement(ManualSubscriber<T> sub, Collection<T> expected) throws InterruptedException {
```

Comment:
Indeed, this is an API that should be as enabling as possible, that means using the most general type from the type hierarchy that has a `contains` method.